huntsman/utils/states.py

```diff
@@ -0,0 +1,20 @@
+from pocs.utils import current_time
+
+
+def past_midnight(pocs):
+    '''
+    Check if is morning, useful for going into either morning or evening flats.
+    '''
+    # Get the time of the nearest midnight to now
+    midnight = pocs.observatory.observer.midnight(current_time(), which='nearest')
+
+    # If the nearest midnight is in the past, its the morning...
+    return midnight < current_time()
+
+
+def wait_for_twilight(pocs):
+    '''
+    Wait for the Sun to be between flat and focus horizons.
+    '''
+    while pocs.is_dark(horizon='focus') or not pocs.is_dark(horizon='flat'):
+        pocs.sleep(delay=pocs._safe_delay)
```

Comment:
A `pocs` instance shouldn't really ever appear anywhere outside of the state files.  Not that it necessarily couldn't, but it isn't how we have organized things. These should go alongside the other "Safety Methods" in `core.py`. We don't override anything in `core.py` for `huntsman-pocs` yet, but there's no reason not to.

(There's a case to be made for moving all of those methods out and doing them in smarter fashion, perhaps via some utils, but that's beyond the scope of this PR).